didcomm/common/types.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+from typing import Dict, Any, Union, List
+
+JSON_DATA = Union[Dict[str, Any], List[Any]]
+JSON = str
+JWK = str
+JWT = str
+JWS = Dict[str, Any]
```

Comment:
Do we need JWS, JSE as dicts? Seems what we will send/receive is serialized to string.